### DISCUSSION TITLE
Incorporate Government Domestic Budget Data into the DDW

### DISCUSSION THREAD

**Discussion by akmiller01** [devinit/ddw-analyst-ui#439] [Q&A]
Add domestic.csv to manual datasets, plus requisite metadata.

**Comment by k8hughes** [comment]
@akmiller01  Is this something where the data is available?

**Comment by akmiller01** [comment]
It's something from the old DDW. Worth asking whether it's still needed by analysts, I suppose.

**Comment by k8hughes** [comment]
@Duncan-Knox what are your thoughts on this?  Do we need this data and is it still being created?

**Comment by Duncan-Knox** [comment]
Domestic.csv is not really being updated any more. I expect we'll end up with a domestic revenue data set at some stage if it is needed on a project. This could be added on the DDW - it won't be the same as domestic.csv and it is not currently being created however.
